libs/designer/src/lib/core/state/operation/operationMetadataSlice.ts

```diff
@@ -259,7 +259,7 @@ export const operationMetadataSlice = createSlice({
 
       const dependencies = getInputDependencies(inputParameters, rawInputs, swagger);
       if (dependencies) {
-        dependencies.inputs = { ...dependencies.inputs, ...dependencies };
+        state.dependencies[nodeId].inputs = { ...state.dependencies[nodeId].inputs, ...dependencies };
```

Comment:
I think @rllyy97 will want to update this code to be more like later in the file:

![image](https://github.com/Azure/LogicAppsUX/assets/1350074/e09a0a2a-f978-4e8f-bf4e-9474130539da)

However, in the interest of a safe & fast hotfix, I've reverted to the previous code verbatim here.